src/app/dynamic-module-loader/terra-dynamic-module-loader.component.ts

```diff
@@ -35,13 +37,21 @@ export class TerraDynamicModuleLoaderComponent implements AfterViewInit, OnDestr
     {
     }
 
-    ngAfterViewInit()
+    public ngAfterViewInit()
     {
         this._resolvedData = this.inputModule as ModuleWithProviders;
         this.updateComponent();
     }
 
-    ngOnDestroy()
+    public ngOnChanges(changes:SimpleChanges):void
+    {
+        if ( changes.hasOwnProperty('inputInputs') )
+        {
+            this.assignInputProperties();
```

Comment:
müssten hier die veränderten Werte nicht übergeben werden?

Reply:
nein. Beim Aufruf von ngOnChanges wurden die neuen Werte bereits zugeordnet und sind somit über `this` erreichbar